softioc/fields.py

```diff
@@ -32,6 +31,8 @@
     DBF_SHORT: c_int16,
     DBF_USHORT: c_uint16,
     DBF_LONG: c_int32,
+    DBF_INT64: c_int64,
+    DBF_UINT64: c_uint64,  # TODO: Check whether this has same issue as below note
```

Comment:
Don't leave a TODO in!

Reply:
Removed in https://github.com/dls-controls/pythonSoftIOC/pull/161/commits/1d9468fe9dd77b9541cdb6ae57771676afb290b2